docs/source/release_notes.rst

```diff
@@ -7,6 +7,7 @@ Release Notes
         * Fixed classification pipelines to only accept target data with the appropriate number of classes :pr:`3185`
         * Added support for time series in ``DefaultAlgorithm`` :pr:`3177`
         * Standardized names of featurization components :pr:`3192`
+	* Removed empty cell :pr:`3234`
```

Comment:
Could you add some more color here? Something like, "Removed empty cell in text_input.ipynb  :pr:`3234`" would be great!